edgecontext.thrift

```diff
@@ -7,7 +7,21 @@ namespace java com.reddit.edgecontext
 */
 typedef string AuthenticationToken
 
-/** A two-character ISO 3166-1 country code
+/** ISO 639-1 language identifier representing the preferred language for
+the client.
+
+*/
+typedef string LanguageCode
+
+/** ISO 3166-1 region identifier representing the preferred locale region
+for the client. This is the region the client is configured with, and not
```

Comment:
> This is the region the client is configured with

for what?  I think we should be explicit if this is intended exclusively for l10n so that people don't end up using it as a "country selector" for geotargeted features by accident.

Reply:
Updated to be more explicit relating to l10n. Also think combining the language and region helps clear this up.